src/react/src/components/AddLocationData.jsx

```diff
@@ -0,0 +1,336 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { withStyles, withTheme } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import CircularProgress from '@material-ui/core/CircularProgress';
+
+import AppGrid from './AppGrid';
+
+import { openInNewTab } from '../util/util';
+import { authLoginFormRoute, InfoLink, InfoPaths } from '../util/constants';
+
+import MessyIcon from './MessyIcon';
+import PlaylistIcon from './PlaylistIcon';
+import PinDropIcon from './PinDropIcon';
+import RectangleCardFigure from './RectangleCardFigure';
+import SliceCardFigure from './SliceCardFigure';
+import SliceMessyFigure from './SliceMessyFigure';
+import SliceMessyDuoFigure from './SliceMessyDuoFigure';
+import COLOURS from '../util/COLOURS';
+
+const addLocationStyles = theme =>
```

Comment:
Can you move these addLocationStyles to the style file to keep the component file clear?

Reply:
I don't see the needed for it and also in the current project we collect in the same file as used for styling.